var/spack/repos/builtin/packages/lbann/package.py

```diff
@@ -99,11 +116,18 @@ def cmake_args(self):
             ('+seq_init' in spec),
             '-DLBANN_WITH_TBINF=OFF',
             '-DLBANN_WITH_VTUNE=OFF',
-            '-DElemental_DIR={0}/CMake/elemental'.format(
-                spec['elemental'].prefix),
             '-DLBANN_DATATYPE={0}'.format(spec.variants['dtype'].value),
             '-DLBANN_VERBOSE=0'])
 
+        if ('@:0.90' or '@0.95:'):
```

Comment:
@bvanessen only issue I see is here: you'll need to replace with

```
if self.spec.satisfies('@:0.90') or self.spec.satisfies('@0.95:'):
...
```

(and likewise with the `elif`). This section of the code doesn't benefit from the same magic as the `@when` decorator - it will be evaluated as a normal python statement (and nonempty strings evaluate to `True`, so as is this branch will always be taken).

Other than that LGTM - if you can update this before ~5:30 today it'll likely pass CI before I leave and I'll merge it, otherwise I'll take a look on Monday morning.